js/components/dataset/resource/file-form.vue

```diff
@@ -22,7 +28,7 @@
     </span>
     <div class="info-box-content">
         <span class="info-box-text">{{file.name}}</span>
-        <span class="info-box-number">{{file.filesize | filesize}}</span>
+        <span class="info-box-number">{{file.size | filesize}}</span>
```

Comment:
I am pretty sure a linter would have caught this issue — @noirbizarre @vinyll do you know of anything to enforce data models for Vue.js?

Reply:
Not sure that a linter can find that a variable name is removed from the spec on a non-typed variable. File(s) in this component (is)are provided by `fineuploader`. The fact its a `File` object is not indicated anywhere else than the fineuploader documentation :/